x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -46,6 +46,15 @@ interface GetAlertParams {
   index?: string;
 }
 
+export const validFeatureIds = ['apm', 'observability', 'siem'] as const;
+export type ValidFeatureIds = typeof validFeatureIds[number];
+
+export const mapConsumerToIndexName: { [featureId in ValidFeatureIds]: string | string[] } = {
+  apm: '.alerts-observability-apm',
+  observability: '.alerts-observability',
+  siem: ['.alerts-security-solution', '.siem-signals'],
+};
```

Comment:
Can we do this?

```ts
export const mapConsumerToIndexName = {
  apm: '.alerts-observability-apm',
  observability: '.alerts-observability',
  siem: ['.alerts-security-solution', '.siem-signals'],
};

export type ValidFeatureId = keyof typeof mapConsumerToIndexName;
```

Then it would be great if we could move this map to the root of the plugin, maybe in a file called something related to RBAC, with some comments at the top explaining what the key and value represents. I think that should make everyone feel good about moving forward with this, since there will be a lot of changes to that one file as we move forward.

Reply:
I put it in a file called `rbac` in the utils folder I figured utils folder would be good.